web/template/partial/stream/chat/chat.gohtml

```diff
@@ -89,181 +75,219 @@
+            <div class="z-50 absolute w-full px-2 py-1" :class="c.admin ? 'top-24' : 'top-12'">
+                <div class="shadow-lg border rounded-lg bg-white py-2 dark:bg-secondary dark:border-gray-600">
+                    <div class="flex align-middle justify-between px-3 border-b pb-2 dark:border-gray-600"
+                         :class="showPoll && 'border-b pb-2'">
+                        <span class="text-xs uppercase font-semibold my-auto">Poll Result</span>
+                        <button @click="c.poll.result = null;">
+                            <i class="fas fa-close"></i>
+                        </button>
+                    </div>
+                    <div class="px-3 pt-3 pb-3">
+                        <p x-text="c.poll.activePoll.question" class="text-center text-base font-light mb-3"></p>
+                        <template x-for="pollOptionResult in c.poll.result.pollOptionResults"
+                                  :key="pollOptionResult.ID">
+                            <div class="mb-3 pl-3 pr-3">
+                                <div class="flex justify-between pr-2">
+                                    <span class="text-sm flex items-end" x-text="pollOptionResult.answer"></span>
+                                    <span class="text-xs flex items-end"
+                                          x-text="pollOptionResult.votes + ' Votes'"></span>
+                                </div>
+                                <div :style="`width: ${watch.getPollOptionWidth(c.poll.result.pollOptionResults, pollOptionResult)};`"
+                                     class="rounded-full h-4 bg-blue-500/50 dark:bg-indigo-600"></div>
                             </div>
-                            <div :style="`width: ${watch.getPollOptionWidth(c.poll.result.pollOptionResults, pollOptionResult)};`"
-                                 class="rounded-full h-4 bg-sky-400"></div>
-                        </div>
-                    </template>
+                        </template>
+                    </div>
                 </div>
             </div>
         </template>
 
         <!-- Messages -->
-        <div x-show="show"
-             id="chatBox"
-             class="relative h-full w-full overflow-y-scroll overflow-x-hidden px-1 z-40 {{if .IsPopUp}}xl:w-1/2 md:w-3/4 md:mx-auto{{end}}">
+        <div id="chatBox"
+             class="grid gap-y-8 content-start h-full w-full overflow-y-scroll overflow-x-hidden px-4 py-2 z-40">
             {{template "messageTemplate" .}}
         </div>
 
         <!-- Messages -->
-        <div class="z-50 bg-white w-full">
+        <div>
             {{template "chatprompts"}}
-            <form x-show="show" id="chatForm"
-                  class="z-50 sticky w-full bottom-0 inset-x-0 p-2 bg-white dark:bg-secondary 2xl:flex 2xl:justify-between
-              {{if .IsPopUp}} xl:w-1/2 md:w-3/4 md:mx-auto md:border md:rounded-t-lg {{end}}"
+            <form id="chatForm"
```

Comment:
When typing a question mark in the chat or the poll window, the shortcuts window pop up. We could use an additional modifier for the shortcut to prevent this.

Reply:
Added the condition `document.activeElement.tagName !== "INPUT" && document.activeElement.tagName !== "TEXTAREA"` so that when an input[textarea]-field is focused the modal will not be toggled.

Reply:
Ah yes, that makes even more sense.